src/interfaces/hapi.ts

```diff
@@ -37,13 +37,13 @@ import * as hapi from 'hapi';
  * @returns {ErrorMessage} - a partially or fully populated instance of
  *  ErrorMessage
  */
-function hapiErrorHandler(req: hapi.Request, err: {}, config: Configuration) {
+function hapiErrorHandler(req: hapi.Request|undefined, err: {}, config?: Configuration) {
```

Comment:
Since this is a non-exported function, change parameter order to `(err, config?, req?)` for better readability.

Reply:
I reordered the arguments as `(err, req?, config?)` since it is more readable.